src/components/IncidentDetails/index.tsx

```diff
@@ -1,30 +1,32 @@
 import { useEffect, useMemo, useState } from "react";
-import { useForm } from "react-hook-form";
+
```

Comment:
What are the changes in this file for ?

Reply:
There were build errors in `main` branch. Just so happened that this PR and another PR addressed it.
The changes in this file is just ordering done by `Tailwind CSS IntelliSense` vscode extension. Would you prefer to revert?